test/unit/DetectChanges/BCBreak/PropertyBased/PropertyDocumentedTypeChangedTest.php

```diff
@@ -196,6 +206,7 @@ class TheClass {
             'privateDocblockToDifferentDocblock'                                => ['[BC] CHANGED: Type documentation for property TheClass#$privateDocblockToDifferentDocblock changed from int to float'],
             'duplicatePropertyTypesBeingDeduplicatedAreNotBcBreaks'             => [],
             'propertyTypeBeingDuplicatedAreNotBcBreaks'                         => [],
+            'propertyWithComplexDocblockThatCannotBeParsed'                     => [],
```

Comment:
Currently, I'd say this is `mixed` -> `mixed`, so we shouldn't detect a BC break here.